impeller/renderer/backend/gles/render_pass_gles.cc

```diff
@@ -585,14 +585,15 @@ bool RenderPassGLES::OnEncodeCommands(const Context& context) const {
 
   std::shared_ptr<const RenderPassGLES> shared_this = shared_from_this();
   auto tracer = ContextGLES::Cast(context).GetGPUTracer();
-  return reactor_->AddOperation([pass_data,
-                                 allocator = context.GetResourceAllocator(),
-                                 render_pass = std::move(shared_this),
-                                 tracer](const auto& reactor) {
-    auto result = EncodeCommandsInReactor(*pass_data, allocator, reactor,
-                                          render_pass->commands_, tracer);
-    FML_CHECK(result) << "Must be able to encode GL commands without error.";
-  });
+  return reactor_->AddOperation(
```

Comment:
Whats the operation that finally enqueues a non-deferred operation? Is there guaranteed to be one? Otherwise, these operations will be pending in the reactor for a potentially long time.

I'm thinking of texture, pipeline, and perhaps blits also adding non-deferred operations. But nothing jumps out at being always present no matter what.

Reply:
CommandBuffer::Submit also performs a flush: https://github.com/flutter/engine/blob/main/impeller/renderer/backend/gles/command_buffer_gles.cc#L32-L38